pkg/webui/console/containers/device-onboarding-form/index.js

```diff
@@ -113,6 +113,14 @@ const DeviceOnboardingForm = () => {
           delete mac_state.current_parameters
         }
 
+        if (values.authenticated_identifiers) {
+          cleanedValues.ids = {
+            ...cleanedValues.ids,
+            ...values.authenticated_identifiers,
+            device_id: values.target_device_id,
+          }
+        }
```

Comment:
This causes a regression in the manual device onboarding flow, causing `ids.dev_eui` to be set to `""` and `ids.authentication_code` to be set, which doesn't exist in `ids`.

Reply:
I think there are two issues here: 1) `values.authenticated_identifiers` seems to be always truthy as it's an object, but the values `dev_eui` and `authentication_code` can be empty and 2) this sets `authentication_code` from `authenticated_identifiers` in `ids` which is wrong, it should be at most the `dev_eui`.

Reply:
```suggestion
        if (values.authenticated_identifiers.dev_eui) {
          cleanedValues.ids = {
            ...cleanedValues.ids,
            dev_eui: values.authenticated_identifiers.dev_eui,
            device_id: values.target_device_id,
          }
        }
```